.gitignore

```diff
@@ -0,0 +1,13 @@
+venv/
```

Comment:
Should we copy the `.gitignore` from, for example, here: https://github.com/canonical/upload-charm-docs/blob/main/.gitignore, it has a bunch of things relating to Python in it already

Reply:
Thanks, that's a better `.gitignore`, I adjusted it. I used the one generated by `charmcraft` and only added terraform specific ignores.